experiments/india_pv_wind.md

```diff
@@ -24,6 +24,67 @@ Overall MAE is 4.9% on the validation set, and forecasts look overall good.
 
 ## WindNet
 
+### WindNet v2 Meteomatics + ECMWF Model
```

Comment:
Can you move this to a seperate experiment file?
 This means we can easily see what the first experiment was, and this will be v2